device.mk

```diff
@@ -431,13 +431,23 @@ PRODUCT_PACKAGES += \
     ApertureOverlayEverpal \
     ApertureQRScannerOverlayEverpal \
     CarrierConfigOverlayEverpal \
-    FrameworksResOverlayEverpal \
-    SettingsOverlayEverpal \
     SystemUIOverlayEverpal \
     TelephonyOverlayEverpal \
     TetheringConfigOverlayEverpal \
-    WifiOverlayEverpal
-
+    WifiOverlayEverpal \
+
+
+ifeq ($(PRODUCT_NAME),evolution_everpal)
+    PRODUCT_PACKAGES += \
+        EvolutionSettingsResEverpal \
+        EvolutionFrameworksResEverpal \
+        FrameworksResOverlayEverpal \
```

Comment:
why add these here I said use them as base overlay for evox containing all changes including evox one